portfolio/src/main/client/src/components/sections/cooking.ts

```diff
@@ -7,14 +7,34 @@ export interface DishInfo {
   html: string;
 }
 
-export const CookingSection = (info: DishInfo): HTMLElement => {
+// get list of files in res/text/dish folder as webpack context
+const dishContext = require.context('@res/text/dish', false);
+
+// load each file from the context these are markdown files that are
+// transformed into JS objects via frontmatter-markdown-loader, giving the
+// front matter on property `attributes` and the HTML corresponding to the
+// markdown on property `html`.
+
+// type assertion b/c webpack context returns `any` since it does not know
+// the type of the modules
+const dishInfos = dishContext
+  .keys()
+  .map((key) => dishContext(key) as DishInfo);
+
+export const CookingSection = (): HTMLElement =>
```

Comment:
nit: add JSDoc comments for all exported items (same with others in this file)

Reply:
What meaningful documentation can I add to this element? `/** This is the cooking section. */` seems pretty useless, but it doesn't do anything except display the cooking section.